Makefile

```diff
@@ -0,0 +1,19 @@
+DECKSCHRUBBER_TAG := v0.6.0
+DIST_TAG := stable
```

Comment:
Instead of this, I'd just make the dist whatever you pass in to the build step, so something like `package_stretch` would set `stretch` as the dist to use. This also makes it easy to build for multiple dists if needed in the future.